torch/csrc/jit/export.h

```diff
@@ -4,8 +4,12 @@
 
 namespace torch { namespace jit {
 
-std::string ExportGraph(const std::shared_ptr<Graph>& graph,
-                        const std::vector<at::Tensor> & initializers,
-                        int64_t onnx_opset_version);
+using RawDataExportMap = std::unordered_map<std::string, std::string>;
```

Comment:
This needs a comment explaining what the string keys and the string values mean. And also more generally what this is used for.